adding-module.rst

```diff
@@ -159,58 +166,77 @@ modules already included in MicroPython. I will not be covering this in detail.
 Classes
 =======
 
-Let's try to add a class to our module. A class is similar to a module -- it's
-also a C struct with certain fields:
+A class is a C-struct with certain fields, quite similar to a module:
 
 .. code-block:: c
 
-    STATIC const mp_rom_map_elem_t mymodule_hello_locals_dict_table[] = {
-    }
-    STATIC MP_DEFINE_CONST_DICT(mymodule_hello_locals_dict,
-                                mymodule_hello_locals_dict_table);
-
-    const mp_obj_type_t mymodule_hello_type = {
-        { &mp_type_type },
-        .name = MP_QSTR_Hello,
-        .print = mymodule_hello_print,
-        .make_new = mymodule_hello_make_new,
-        .locals_dict = (mp_obj_dict_t*)&mymodule_hello_locals_dict,
+    // creating the table of global members
+    STATIC const mp_rom_map_elem_t mymodule_hello_locals_dict_table[] = { };
+    STATIC MP_DEFINE_CONST_DICT(mymodule_hello_locals_dict, mymodule_hello_locals_dict_table);
+
+    // create the class-object itself
+    const mp_obj_type_t mymodule_helloObj_type = 
+    {
+        { &mp_type_type },                  // "inherit" the type "type"
+        .name = MP_QSTR_helloObj,           // give it a name
+        .print = mymodule_hello_print,      // give it a print-function
+        .make_new = mymodule_hello_make_new,    // give it a constructor
+        .locals_dict = (mp_obj_dict_t*)&mymodule_hello_locals_dict, // and the global members
```

Comment:
Please use consistent spacing with comments: two spaces, not some random indentations